src/app/components/GoogleSheetsInput.svelte

```diff
@@ -107,14 +123,15 @@
   </script>
 </svelte:head>
 
+<UploadAdvancedInput bind:wordColumnInd bind:countColumnInd />
 <div class="google-sheets" data-cy="google-sheets-input">
   {#if error}
     <p class:error>{error}</p>
   {/if}
   <InputField
     label="Google Sheets URL"
     id="googleSheetsURL"
-    bind:value={googleSheetsURL}
+    bind:inputValue={googleSheetsURL}
```

Comment:
Just curious, what would be the difference between having `value` and `inputValue`?

Reply:
Good question! `value` isn't defined as a prop in `<InputField>`, but `inputValue` is. https://github.com/eddieantonio/predictive-text-studio/blob/a7203d9a93c84b04cf1015531ac6d66d4821699c/src/app/components/InputField.svelte#L9-L16.

Not entirely sure why it worked previously

Reply:
😂  My understanding is that the way we're grabbing the URL was by using the `id` prop of the input i.e. `document.getElementById("input-googleSheetsURL")` with `value` never really bound to the input.